src/msw/window.cpp

```diff
@@ -3707,57 +3707,68 @@ wxWindowMSW::MSWHandleMessage(WXLRESULT *result,
         case WM_NCCALCSIZE:
             {
                 const wxBorder border = TranslateBorder(GetBorder());
+
                 if (wxUxThemeIsActive() && border == wxBORDER_THEME)
                 {
+                    RECT newrcClient; // new client rect
+
                     // first ask the widget to calculate the border size
                     rc.result = MSWDefWindowProc(message, wParam, lParam);
                     processed = true;
 
-                    // now alter the client size making room for drawing a
-                    // themed border
-                    RECT *rect;
-                    NCCALCSIZE_PARAMS *csparam = NULL;
-                    if ( wParam )
-                    {
-                        csparam = (NCCALCSIZE_PARAMS *)lParam;
-                        rect = &csparam->rgrc[0];
-                    }
-                    else
+                    if (wParam)
                     {
-                        rect = (RECT *)lParam;
+                        LPNCCALCSIZE_PARAMS pnccsp = (LPNCCALCSIZE_PARAMS)lParam;
+                        // pnccsp->rgrc[0] is the computed new client rect
+                        // pnccsp->rgrc[1] is going to be the dst blit rect
```

Comment:
Is this really relevant as long as we never return `WVR_VALIDRECTS` anyhow?

Reply:
No, I left the comments to remind me of the signification of the `NCCALCSIZE_PARAMS` values  but I forgot to remove them, I am going to remove'em.